src/nwm_routing/src/nwm_routing/__main__.py

```diff
@@ -1198,6 +1204,7 @@ def main_v03(argv):
             compute_kernel,
             subnetwork_target_size,
             cpu_pool,
+            t0,
             dt,
             nts,
             qts_subdivisions,
```

Comment:
After run_results is returned can we just copy over the values found in usgs or usace? There is only a fraction of the timestep locations, but this copies over the first 0:n timesteps. Possible approach.


#find the location of the reservoir ids in order to index the results values. 
  usgs_idx=np.where(np.isin(run_results[0][0], [reservoir_usgs_df.index]))[0]
  usace_idx=np.where(np.isin(run_results[0][0], [reservoir_usace_df.index]))[0]
  #loop through values and replace n timesteps with da values. 
  for location, value in enumerate(usgs_idx):
      run_results[0][1][value][:len(reservoir_usgs_df.values[location])] = reservoir_usgs_df.values[location]
  for location, value in enumerate(usace_idx):
      run_results[0][1][value][:len(reservoir_usace_df.values[location])] = reservoir_usace_df.values[location]